src/config/urls.js

```diff
@@ -64,6 +64,7 @@ export const urls = {
     "https://support.ledger.com/hc/en-us/articles/360021039173-Choose-network-fees?docs=true",
   recipientAddressInfo:
     "https://support.ledger.com/hc/en-us/articles/4404389453841-Receive-crypto-assets?docs=true",
+  managerAppLearnMore: "https://support.ledger.com/hc/en-us/categories/4404376139409?docs=true",
```

Comment:
Is the link correct? It's valid but it directs me to the generic welcome page of Ledger Docs.

Reply:
It's what I got, yes 🤷🏼